tests/queries/0_stateless/02864_statistic_uniq.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS t1;
+
+SET allow_experimental_statistic = 1;
+SET allow_statistic_optimize = 1;
+
+CREATE TABLE t1
+(
+    a Float64 STATISTIC(tdigest),
```

Comment:
So `Uniq` can only be used in combination with tdigest?

Reply:
right now the answer is yes. Because it is only used when estimating `=` conditions when the cardinality is low. But this method is relied on `TDigest`